cpp/libcudf_kafka/CMakeLists.txt

```diff
@@ -147,7 +147,13 @@ include_directories("${RDKAFKA_INCLUDE}")
 
 ###################################################################################################
 # - cudf_kafka Install ----------------------------------------------------------------------------
-target_link_libraries(cudf_kafka cudf)
+
+# add third party dependencies using CPM
+include(cmake/thirdparty/CUSPATIAL_GetCPM.cmake)
+# find cuDF
+include(cmake/thirdparty/CUSPATIAL_GetCUDF.cmake)
```

Comment:
Copy paste error?

Reply:
Yes, got in far too much of a hurry. I'm about to make another commit cleaned up